php/php.dbgp/src/org/netbeans/modules/php/dbgp/breakpoints/BreakpointsReader.java

```diff
@@ -34,14 +34,16 @@ public class BreakpointsReader implements Properties.Reader {
 
     private static final String ENABED = "enabled"; // NOI18N
     private static final String FUNC_NAME = "functionName"; // NOI18N
+    private static final String EXCEPTION_NAME = "exceptionName"; // NOI18N
     private static final String TYPE = "type"; // NOI18N
     private static final String GROUP_NAME = "groupName"; // NOI18N
 
     @Override
     public String[] getSupportedClassNames() {
         return new String[]{
             LineBreakpoint.class.getName(),
-            FunctionBreakpoint.class.getName()
+            FunctionBreakpoint.class.getName(),
+            ExceptionBreakpoint.class.getName()
```

Comment:
Maybe, should move to a field?
e.g. 
```java
private static final String[] SUPPORTED_CLASS_NAMES = ...;

public String[] getSupportedClassNames() {
        return SUPPORTED_CLASS_NAMES;
}    
```

Reply:
Or remove it and make `SUPPORTED_CLASS_NAMES` `public`?

Reply:
I can't remove the `getSupportedClassNames` method as it is an implementation of the `org.netbeans.api.debugger.Properties.Reader` interface. Did as @junichi11  suggested.

Reply:
I see. In such a case, please add `@Override` annotation. Thanks!